src/main/resources/db/migration/V1_6_0__desired_outcome.sql

```diff
@@ -0,0 +1,16 @@
+create table desired_outcome(
+  id uuid not null,
+  description text not null,
+  service_category_id uuid not null,
+
+  primary key (id),
+  constraint fk__desired_outcome__service_category foreign key(service_category_id) references service_category
+);
+
+create table referral_desired_outcomeids(
+    referral_id        uuid not null,
+    desired_outcomeids uuid,
+
+    constraint fk__referral_desired_outcomeids__referral foreign key (referral_id) references referral
```

Comment:
Given this table will be used to look up outcomes by referral ID, does this need an index on the `referral_id`?

Reply:
yes, it was created from the code hibernate generates.  It also needs to be not null.  I will update and push back - also renamed to remove the ids part of the name that was also automatically created

Reply:
Will add in the index separately